SWLOR.Game.Server/Feature/ScavengePoint.cs

```diff
@@ -113,6 +113,10 @@ public static void OnDisturbed()
                 if (!GetIsObjectValid(firstItem))
                 {
                     DestroyObject(placeable);
+
+                    // DestroyObject bypasses the OnDeath event, and removes the object so we can't send events.
```

Comment:
Re: previous comment - this is what I want to avoid :) Definitely feel strongly about moving the ExecuteScript call in the overridden DestroyObject function after seeing what you've done here.